graviti/manager/commit.py

```diff
@@ -44,36 +46,36 @@ class Commit(Sheets, AttrsMixin):
 
     """
 
-    _repr_attrs: Tuple[str, ...] = ("parent_commit_id", "title", "committer", "committed_at")
+    _commit_info: Dict[str, Optional[str]]
```

Comment:
shall we put `_repr_attrs` above `_commit_info` due to `_repr_attrs` is a class variable?